babel.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  presets: ['@babel/preset-env', '@babel/preset-react'],
+  plugins: [
+    '@babel/plugin-proposal-class-properties',
+    [
+      'module-resolver',
+      {
+        alias: {
+          'cypress-react-unit-tests': './dist'
```

Comment:
nice

Reply:
I just noticed `cypress-react-unit-tests` - and changed it to `cypress-react-unit-test` :)